### PR TITLE
Update of functions for salt and pepper with data type = float

### PR DESCRIPTION
**Pull request by KonanAl**
These versions of saltAndPepper3dNoise_D and saltAndPepper2dNoise_D have been checked together and also tested

### CODE REVIEW

File: ImageInLib/ImageInLib/include/common_functions.h [hunk 1]
```diff
@@ -45,6 +45,15 @@ extern "C" {
...
 	} Image_Data;
+
+	// Generate Random Points
+	typedef struct {
+		size_t k, xd, p;
+	}RandomPoints;
+
+	typedef struct {
+		size_t xd, p;
+	}Random2dPoints;
```
Comment (deaspo) [inline ImageInLib/ImageInLib/include/common_functions.h:56]: Give a more clear name i.e. Random3DPoints and Random2DPoints or just use one general structure RandomPoints for both cases.

**Comment by KonanAl** [comment]
new commit with updates for RandomPoints

**Comment by deaspo** [comment]
Why did you close?

**Review by deaspo — COMMENTED** [review]
Looks okay now but it is closed

**Pull request closed without merging**